src/org/loklak/harvester/RedirectUnshortener.java

```diff
@@ -6,12 +6,12 @@
  *  modify it under the terms of the GNU Lesser General Public
  *  License as published by the Free Software Foundation; either
  *  version 2.1 of the License, or (at your option) any later version.
- *  
+ *
```

Comment:
Unwanted change This is a standard template that gets prepended to the start of the file and needn't be modified.